l10n_fr_department/model/res_partner.py

```diff
@@ -10,16 +10,19 @@
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
-    department_id = fields.Many2one(
+    # In v16, this field was renamed from department_id to country_department_id
+    # to avoid a field name conflict with the module partner_contact_department
+    # from https://github.com/OCA/partner-contact
```

Comment:
These comments belong to the commit message or to the PR itself.

They're only useful when reviewing/analyzing changes from previous code.
But they have no use at all in the present/future code